oot_debug_syms.ld

```diff
@@ -3309,3 +3309,4 @@ gScreenWidth = 0x8012D270;
 gScreenHeight = 0x8012D274;
 gMtxClear = 0x8012DB20;
 gTimeSpeed = 0x8011FB40;
+sCurrentMatrix = 0x8016A7C4;
```

Comment:
If this thing was originally static isn't it a bit weird that it is being manipulated globally?

Reply:
the "correct" thing to do would be to access via Matrix_ functions, `Matrix_Get` and `Matrix_Put` for example

Reply:
yeah but it's used to add Matrix_ functions that, if in decomp, would be added in sys_matrix and use the symbol directly

the one use in `Physics_SetHead` is me thinking about performance